pkg/ibmvpc/examples/acl_testing5subnetsDiff.json

```diff
@@ -0,0 +1,212 @@
+{
+    "semantic_diff": [
+        {
+            "diff-type": "connection changed",
+            "src": {
+                "ResourceName": "sub1-1-ky",
+                "ResourceUID": "crn:17",
+                "ResourceType": "Subnet",
+                "Zone": "us-south-1"
+            },
+            "dst": {
+                "ResourceName": "sub1-2-ky",
+                "ResourceUID": "crn:36",
+                "ResourceType": "Subnet",
+                "Zone": "us-south-1"
+            },
+            "conn1": [
+                {
+                    "protocol": "TCP"
+                }
+            ],
+            "conn2": [
+                {
+                    "protocol": "TCP"
+                }
+            ]
+        },
```

Comment:
why is this `connection changed` if both `conn1` and `conn2` are the same?

Reply:
Indeed, in my tests I somehow missed it. 
The changed segment is the statefullness.  Why isn't `ConnToJSONRep` considering this aspect? Perhaps we should have this fix in another PR (and delay the merge of README until that PR is ready)

Reply:
in the link for this issue, seems it was opened in the wrong repo (`vpc-network-config-synthesis ` instead of this repo)

Reply:
https://github.com/np-guard/vpc-network-config-analyzer/issues/240